### DISCUSSION TITLE
Odd off topic question: buy a frequency range

### DISCUSSION THREAD

**Discussion by mariotti** [meshtastic/firmware#5278] [Ideas]
While designing my app, I had to read different regulations.
As I am in EU, this is what I was checking.

For example to use LoRa EU868, I need to comply to a duty cycle of 1%, which is about few seconds within an hour.
Due to the specific of Lora and standard applications this is relatively acceptable especially because it is defined
on a per device base.

But this constrain would be released, if the frequency is not public.

In theory if a we (a cooperative, a company, an org) would buy a frequency range,
we would not have this limitation.

I assume that then the cooperative/company/org will put back an equivalent limitation.

Bu it would be handled by project, and not by state regulation. 

On the other side, you risk to be out of law even for small local projects.

Is there any person here, that can consider this option?

I was looking for contacts, but could not find any.

Thanks in advance,

Fabio

**Comment by fifieldt** [comment]
A random price I saw for spectrum this year was: $0.106/MHz/pop

So, for example, for Italy: 7MHz * $0.106 * 58.76 million = $43,599,920

**Reply by mariotti** [reply]
Can you give a reference.

The numbers makes sense, indeed,  if you want your private network, but what about a social project?
I am pretty sure these numbers are much lower.

btw: thanks!

I am indeed from IT too, nice match.

Thanks

**Comment by mariotti** [comment]
I close the discussion as probably is not yet an option ;)
